Framework/DataHandling/src/SaveGSS.cpp

```diff
@@ -833,11 +987,26 @@ void SaveGSS::writeSLOGdata(const int bank, const bool MultiplyByBinWidth,
   g_log.debug() << "SaveGSS(): Min TOF = " << bc1 << '\n';
 
   // Write bank header
-  writeBankHeader(out, "SLOG", bank, datasize);
-  out << std::fixed << " " << std::setprecision(0) << std::setw(10) << bc1
-      << std::fixed << " " << std::setprecision(0) << std::setw(10) << bc2
-      << std::fixed << " " << std::setprecision(7) << std::setw(10) << bc3
-      << std::fixed << " 0 FXYE\n";
+  if (overwrite_std_bank_header_) {
```

Comment:
It doesn't look like the logic matches the documentation. Should it be
```
if (!user_specified_bank_headers_.empty())
  // write it out

if (!overwrite_std_bank_header_)
  // write it out
```
The current implementation cannot have both bank headers.

Reply:
The logic is to use either user-specified header or standard header.